.github/workflows/docker-publish.yml

```
@@ -91,8 +101,20 @@ jobs:
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+  multiarch:
+    needs: [ build, preBuild ]
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    steps:
+        - uses: Noelware/docker-manifest-action@v1
+          with:
+              inputs: ${{ env.IMAGE_NAME }}:latest,${{ env.IMAGE_NAME }}:${{needs.preBuild.tag}}
+              images: ${{ env.IMAGE_NAME }}:${{needs.preBuild.tag}}-amd64,${{ env.IMAGE_NAME }}:image/${{needs.preBuild.tag}}-arm64
```

actionlint (1.7.4)

property "tag" is not defined in object type {outputs: {commit: string; tag: string}; result: string}

(expression)